coalib/bearlib/abstractions/Linter.py

```diff
@@ -368,6 +379,37 @@ def process_output_corrected(self,
                              diffs={filename: diff},
                              severity=diff_severity)
 
+        def process_output_unidiff(self,
+                                   output,
+                                   filename,
+                                   file,
+                                   diff_severity=RESULT_SEVERITY.NORMAL,
+                                   result_message='Inconsistency found.'):
```

Comment:
Actually it's possible to disable diff-splitting by setting `diff-distance` to `0` or `-1`. So yeah let's introduce `diff-distance` :+1: